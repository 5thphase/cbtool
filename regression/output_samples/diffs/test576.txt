 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
 status: AI xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx was successfully undefined on SimCloud "TESTCLOUD".
|Redis Version |2.8.6 
|HOSTs |60 
|VMCs ISSUED |16 
|HOSTs ISSUED |0 
|VMs ISSUED |116 
|VMs SLA PROVISIONING VIOLATED |0 
|VMs SLA RUNTIME VIOLATED |0 
|VMs APPLICATION ERRORS |7 
|AIs ISSUED |29 
|AIDRSs ISSUED |0 
|VMCRSs ISSUED |0 
|FIRSs ISSUED |0 
|EXPERIMENT COUNTER |494 
