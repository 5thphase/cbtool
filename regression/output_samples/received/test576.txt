 status: Sending a termination request for vm_74 (cloud-assigned uuid A2432D01-A842-5417-8DA3-266F6CCF7DEA)....
 status: Sending a termination request for vm_50 (cloud-assigned uuid 8CDBDF8E-9887-5699-88BA-92FA67589554)....
 status: Sending a termination request for vm_47 (cloud-assigned uuid 05E9F0C4-DAFB-56C5-BD4F-6DA9A77D6F94)....
 status: Sending a termination request for vm_48 (cloud-assigned uuid 648BDB28-B47A-5FCB-836F-F837FD21FB06)....
 status: Sending a termination request for vm_73 (cloud-assigned uuid 1D631771-47BB-5DC2-B2FB-6AF05B36B90F)....
 status: Sending a termination request for vm_49 (cloud-assigned uuid 7239D9A7-D6F7-5334-93EF-1DC3259093E3)....
 status: Sending a termination request for vm_51 (cloud-assigned uuid 3A2C44F5-F022-5922-8044-323C9A53780C)....
 status: Sending a termination request for vm_59 (cloud-assigned uuid 75FFD9B6-2BCD-505E-AC31-996F7BE8678D)....
 status: Sending a termination request for vm_58 (cloud-assigned uuid 380B6DB2-2C78-5B69-A468-99B3ED2F259B)....
 status: Sending a termination request for vm_60 (cloud-assigned uuid 8D1E6ACF-0578-53ED-83C4-183EE28F860A)....
 status: Sending a termination request for vm_76 (cloud-assigned uuid 4B353D68-0C65-555D-B042-22E1CEB983F5)....
 status: Sending a termination request for vm_92 (cloud-assigned uuid 8D5FED87-DEC7-5302-93D1-51B0FC9F0E29)....
 status: Sending a termination request for vm_95 (cloud-assigned uuid 2F0FBD67-2F71-5F9E-AC0D-9F175705709A)....
 status: Sending a termination request for vm_75 (cloud-assigned uuid 1E632F0C-F534-5BA5-8972-FAFA423CFF6B)....
 status: Sending a termination request for vm_93 (cloud-assigned uuid EC58C761-CF99-5920-BBFC-A50227922972)....
 status: Sending a termination request for vm_96 (cloud-assigned uuid 2F061335-02E3-583C-906B-A26BFA72C73C)....
 status: Sending a termination request for vm_52 (cloud-assigned uuid 4A894F73-62D0-586D-9BD2-D00118EE04FB)....
 status: Sending a termination request for vm_54 (cloud-assigned uuid ED156EA1-B2EF-5DB5-9F78-A8013A3D3BE3)....
 status: Sending a termination request for vm_53 (cloud-assigned uuid 6E175004-81F0-5066-9621-A435257BF2C0)....
 status: Sending a termination request for vm_94 (cloud-assigned uuid FBAAC002-8BED-5DFD-9000-FA7AAA5117C6)....
 status: Sending a termination request for vm_108 (cloud-assigned uuid 636D3B60-4285-5848-8A8B-A00E45A8D6A8)....
 status: Sending a termination request for vm_77 (cloud-assigned uuid BA775660-45DA-56F4-AE90-7748B2F82762)....
 status: Sending a termination request for vm_78 (cloud-assigned uuid 30E82476-480F-5AB4-BA49-43E145FA86E4)....
 status: Sending a termination request for vm_112 (cloud-assigned uuid 307F1EEE-4117-5B71-B1CF-0B60BB9B7B17)....
 status: Sending a termination request for vm_71 (cloud-assigned uuid C544E5A1-5994-58BC-A685-E678C02779CB)....
 status: Sending a termination request for vm_109 (cloud-assigned uuid 69FBE4A2-6EFD-57BE-8919-D1261A187D44)....
 status: Sending a termination request for vm_110 (cloud-assigned uuid 57C906D2-D33E-534C-85E8-D4E18D2963E9)....
 status: Sending a termination request for vm_111 (cloud-assigned uuid 7290A4D1-EDA9-5658-922D-4CE9B6DE1563)....
 status: Sending a termination request for vm_97 (cloud-assigned uuid 3AD080DE-87FF-5391-9EE7-D5FCB94B32B0)....
 status: Sending a termination request for vm_72 (cloud-assigned uuid A1BB7061-B94C-5990-B0BA-1A0B77B8E07F)....
 status: Sending a termination request for vm_84 (cloud-assigned uuid FB7895B7-0B07-5DDB-A2B1-7C5DCE6397CE)....
 status: Sending a termination request for vm_98 (cloud-assigned uuid 5984D0E1-ABE6-5D10-84A1-008DABF51F26)....
 status: Sending a termination request for vm_85 (cloud-assigned uuid D657FAAB-A151-50DD-BE7D-125E3314ED33)....
 status: Sending a termination request for vm_104 (cloud-assigned uuid 2B82CE8D-88F4-51C3-B98A-CD379CD80FB1)....
 status: Sending a termination request for vm_103 (cloud-assigned uuid E90ACEFC-B53C-58CC-9F91-EF2836A6DD9B)....
 status: Sending a termination request for vm_105 (cloud-assigned uuid 49A90D99-8301-50DD-A063-0B27C8D9C364)....
 status: Sending a termination request for vm_106 (cloud-assigned uuid 6E427B07-AA9B-5989-8031-C78EFEA1AE1B)....
 status: Sending a termination request for vm_107 (cloud-assigned uuid A952F6B4-3A9D-562D-95A6-2F7D57273E40)....
 status: Sending a termination request for vm_79 (cloud-assigned uuid B5E97028-35EE-5B46-84E2-F2B406C0A19D)....
 status: Sending a termination request for vm_80 (cloud-assigned uuid 69A5B102-A722-525D-B3E0-028357A234C5)....
 status: Sending a termination request for vm_82 (cloud-assigned uuid 46BBE8CB-5B1F-51DE-A055-2F086C653FDA)....
 status: Sending a termination request for vm_39 (cloud-assigned uuid 0776870C-7FCE-54D9-B339-86DF2AB0EA7E)....
 status: Sending a termination request for vm_113 (cloud-assigned uuid 00C8E3E1-4708-5F5F-A9A4-9B8A8E340EF6)....
 status: Sending a termination request for vm_81 (cloud-assigned uuid 1566ACA7-9396-546F-8033-917C6806EE3F)....
 status: Sending a termination request for vm_86 (cloud-assigned uuid 4751F566-F165-59BE-814A-FCE8EEACEFA8)....
 status: Sending a termination request for vm_40 (cloud-assigned uuid 319481DB-4B25-5D3F-A766-261A4E1C416C)....
 status: Sending a termination request for vm_116 (cloud-assigned uuid 905B62F4-3E4B-5873-8C1F-D0AE6D133812)....
 status: Sending a termination request for vm_115 (cloud-assigned uuid 421F161E-A0DE-5A76-A1B8-F03334375B33)....
 status: Sending a termination request for vm_83 (cloud-assigned uuid 75F452D5-3049-5C4A-B222-D8B64053857C)....
 status: Sending a termination request for vm_87 (cloud-assigned uuid 63F10210-FAF3-5CCA-AE85-A41C5688B597)....
 status: Sending a termination request for vm_88 (cloud-assigned uuid 53DF203C-5315-552C-B7AE-9AE6706222BF)....
 status: Sending a termination request for vm_114 (cloud-assigned uuid 33D68318-CECB-5AC9-9911-E976081D7D2A)....
 status: Sending a termination request for vm_89 (cloud-assigned uuid A36EF480-AD8A-5BE9-8EBA-9A84DF539EDA)....
 status: AI 4F8C3431-9280-5DFD-98D0-D4620D62756D was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 49B005B9-BEFB-561A-960B-0D1CD7ADC1A8 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 479F4FC5-47F3-53CF-AB3D-8562DE47451B was successfully undefined on SimCloud "TESTCLOUD".
 status: AI CEEFE949-2F57-5D0B-BF78-B201F2266A19 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI FF8CD774-D395-5789-A48F-7ABC6C0E6863 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI C2FD9A64-6BAB-5C67-9CE1-84FADF4C496F was successfully undefined on SimCloud "TESTCLOUD".
 status: AI D58A4EA0-3B5D-59BC-903E-D0B460142D67 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI F52B9735-2EB0-5198-A35F-8726AE289A8C was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 864E890E-CDDB-5712-9AAF-2E49ABB14521 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI DAC168F6-82EB-55C2-998B-C0BC2B3C9691 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI F8D514E7-9B5D-5910-9B2E-F1E578ACB317 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 229B6C7E-006B-5944-BC34-6AA5A93721F6 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI C8A33C49-EF10-5922-8B9A-6658D917CC84 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI FE97C596-A2D9-5869-9F9B-6677FDC0B9C6 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 024ED90E-F0F9-57CC-80DA-09D73EEF4F90 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 8147C7AF-8722-58E8-A192-F15E7931760B was successfully undefined on SimCloud "TESTCLOUD".
AI object undefined (named "all") was sucessfully detached from this experiment.
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
No objects available.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
No objects available.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |17.69M                        
|Redis Version                                        |3.0.6                         
|Uptime (in seconds)                                  |8705872                       
|Total Connections Received                           |377546                        
|Total Commands Processed                             |1223806918                    
|Number of Keys                                       |1268                          
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |15                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ISSUED                                           |116                           
|VMs ARRIVED                                          |113                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |113                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |6                             
|AI RESERVATIONS                                      |0                             
|AIs ISSUED                                           |29                            
|AIs ARRIVED                                          |28                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |28                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |1                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |494                           

 ---------------- VMC-WIDE COUNTERS ----------------
|9E74D721-5227-5C9B-A7BD-BE5B4A43D6AA (simzone_d) VM RESERVATIONS|0                             
|3B6C80F6-6318-5721-A484-E3DA77718B57 (simzone_c) VM RESERVATIONS|0                             
|803C5D90-3C59-5871-9ACF-5683E3FB4506 (simzone_a) VM RESERVATIONS|0                             
|270A5605-A7C0-5F5E-B66F-19B77C6645BE (simzone_b) VM RESERVATIONS|0                             

