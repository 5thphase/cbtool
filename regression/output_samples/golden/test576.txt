 status: Sending a termination request for vm_54 (cloud-assigned uuid CFB6E68E-F25F-5BDA-8294-8919C55299ED)....
 status: Sending a termination request for vm_53 (cloud-assigned uuid 94E978A9-E7AF-5AB7-9404-83A2A8957FF5)....
 status: Sending a termination request for vm_52 (cloud-assigned uuid 1AB56039-A236-5BA6-A44D-919E05657046)....
 status: Sending a termination request for vm_86 (cloud-assigned uuid A11450A1-B9A6-5A59-871E-E0EF0A64C5F2)....
 status: Sending a termination request for vm_89 (cloud-assigned uuid B1B832FB-5E3F-5CD6-AB87-C324D9213874)....
 status: Sending a termination request for vm_87 (cloud-assigned uuid 68DA77A3-FE36-5F0B-9909-4226E8542158)....
 status: Sending a termination request for vm_88 (cloud-assigned uuid 6223C019-C70D-5D6F-99D5-CE960C5F2C11)....
 status: Sending a termination request for vm_108 (cloud-assigned uuid 85B92618-26DE-5A1E-81D2-7050205D3C7A)....
 status: Sending a termination request for vm_111 (cloud-assigned uuid 99BD41CB-FA09-52FA-A9D4-8C94A7D41D6D)....
 status: Sending a termination request for vm_109 (cloud-assigned uuid DB917F96-8391-5807-929E-18D4445499B3)....
 status: Sending a termination request for vm_98 (cloud-assigned uuid 895E6099-850D-5E5B-8996-E0D6BA36C236)....
 status: Sending a termination request for vm_97 (cloud-assigned uuid 48DED2E8-CC03-5A02-891D-709524EAD06E)....
 status: Sending a termination request for vm_40 (cloud-assigned uuid 2F2424BB-CF26-5898-BFC7-C93AC9353661)....
 status: Sending a termination request for vm_112 (cloud-assigned uuid 92EE1B33-91CB-5102-8361-FCACB879AD0B)....
 status: Sending a termination request for vm_84 (cloud-assigned uuid D52C4707-710C-5CF1-996D-4117D99519CA)....
 status: Sending a termination request for vm_39 (cloud-assigned uuid 89FE2612-F459-519E-B109-8563A3052F1A)....
 status: Sending a termination request for vm_110 (cloud-assigned uuid 2E52396B-504B-5DEE-9198-E4D8BAE8B6DF)....
 status: Sending a termination request for vm_79 (cloud-assigned uuid 64557C50-9364-5D8E-BDBE-D59BD39C5D69)....
 status: Sending a termination request for vm_81 (cloud-assigned uuid 5693ECD0-F58A-50D6-8F0B-988A5FBDE60A)....
 status: Sending a termination request for vm_85 (cloud-assigned uuid 2F045B59-BE01-556A-9659-BD9073277EF9)....
 status: Sending a termination request for vm_80 (cloud-assigned uuid 8AD66205-434A-5876-B704-5B4EF8607870)....
 status: Sending a termination request for vm_82 (cloud-assigned uuid 99C5CF6E-9BEB-5093-855B-7D9B8FB6F595)....
 status: Sending a termination request for vm_83 (cloud-assigned uuid 02EE020E-F038-5FA3-9EF8-28E15E85E14B)....
 status: Sending a termination request for vm_90 (cloud-assigned uuid BA1222C8-9B82-5956-A074-8838A247326D)....
 status: Sending a termination request for vm_58 (cloud-assigned uuid AB1131F7-C274-5A2D-BBDC-3E75CB6929B1)....
 status: Sending a termination request for vm_59 (cloud-assigned uuid BBD15A96-407E-5849-B2D7-1CE5217758E2)....
 status: Sending a termination request for vm_60 (cloud-assigned uuid 580ED292-516F-55F5-9774-47AA1EC5DECF)....
 status: Sending a termination request for vm_75 (cloud-assigned uuid 702B8AB4-BAE8-54BF-84D5-1548E8215C7E)....
 status: Sending a termination request for vm_74 (cloud-assigned uuid 24C2817E-C58B-5065-BABD-16E320957C05)....
 status: Sending a termination request for vm_76 (cloud-assigned uuid 2528A549-A50B-526F-9B00-09516154A720)....
 status: Sending a termination request for vm_73 (cloud-assigned uuid A9F1DB5A-C254-596E-8897-8428CB2B53AC)....
 status: Sending a termination request for vm_71 (cloud-assigned uuid 11643B02-613A-57EC-B6E7-4052B2E42E30)....
 status: Sending a termination request for vm_72 (cloud-assigned uuid 84B8F9B9-7E1F-524F-9D0D-A78D8C56A89A)....
 status: Sending a termination request for vm_93 (cloud-assigned uuid 082B694A-0539-5591-9801-36505E0932D7)....
 status: Sending a termination request for vm_92 (cloud-assigned uuid 4AD3D558-634F-561C-989E-90B7879CF9C3)....
 status: Sending a termination request for vm_94 (cloud-assigned uuid 176A8D1F-A398-55C2-9380-AE8734C23471)....
 status: Sending a termination request for vm_95 (cloud-assigned uuid B75608C2-71AA-5C38-AB26-EA2D37368A11)....
 status: Sending a termination request for vm_96 (cloud-assigned uuid 2A379CAB-6A0C-5D72-B04B-0701AAE2A7B9)....
 status: Sending a termination request for vm_113 (cloud-assigned uuid 17097039-8263-5E3F-87BB-03B11E462337)....
 status: Sending a termination request for vm_114 (cloud-assigned uuid B67572C7-4C0F-5818-B4FF-8AD1921EE0BE)....
 status: Sending a termination request for vm_115 (cloud-assigned uuid 297633E7-2A0B-5FA7-9B2B-A9F48397FB23)....
 status: Sending a termination request for vm_77 (cloud-assigned uuid 9B249769-DF41-5D81-A65E-13BB1DB0EEC7)....
 status: Sending a termination request for vm_103 (cloud-assigned uuid 1445B8B4-21D4-56B3-8DFC-E9648D5FD39C)....
 status: Sending a termination request for vm_116 (cloud-assigned uuid A959591E-63CE-55C6-BF42-1F40E5E13355)....
 status: Sending a termination request for vm_104 (cloud-assigned uuid 62B02E8E-9831-514E-9CB7-76E6E2965957)....
 status: Sending a termination request for vm_78 (cloud-assigned uuid 49241729-71F8-5BBA-82B9-F654032C84E2)....
 status: Sending a termination request for vm_105 (cloud-assigned uuid 6E43D8F2-C31E-5C75-84B2-AC59DF1BF6F2)....
 status: Sending a termination request for vm_106 (cloud-assigned uuid 562F6542-6891-5BE6-8238-C9152605448E)....
 status: Sending a termination request for vm_107 (cloud-assigned uuid 1403CA33-8CEF-5155-9BC1-888CA1255B29)....
 status: AI 02762095-2332-529F-9BF6-038963923593 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI A0623B90-34D9-58B7-B348-436409D9EB1E was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 6BE003BA-0A4C-5AB6-9880-EED26B351945 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 12C2B91F-493E-5D4D-8259-C778090F7EB5 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 3A6D6FDF-8313-5459-AB01-3A03954CA3A4 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI C0637DB8-4E80-5F9E-BD39-775113089F76 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 1088C26B-B699-5F92-86E0-4EABB7C21A2A was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 8B6E330D-ABC8-5AC1-9F2F-7BE8CB3EBD57 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 48072647-2BB3-5580-A212-2562CDD771F3 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 92F28A12-4E6F-5EC8-8D34-0BC14236FC3D was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 1401409F-4D49-5078-A27F-36448A8C1514 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI D8E11067-DB15-501D-8215-09143E2CFF77 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI FB25FD43-608D-5757-96B6-B69D340C9B0F was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 2907783D-742F-5464-B0F6-AC9B6E89B2C6 was successfully undefined on SimCloud "TESTCLOUD".
 status: AI 3848381E-5461-53C1-82D2-C7C1459264BD was successfully undefined on SimCloud "TESTCLOUD".
 status: AI A7E92E2F-A83E-550C-A3F9-F50C8362062E was successfully undefined on SimCloud "TESTCLOUD".
AI object undefined (named "all") was sucessfully detached from this experiment.
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
No objects available.
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
No objects available.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |23.27M                        
|Redis Version                                        |2.8.6                         
|Uptime (in seconds)                                  |12857448                      
|Total Connections Received                           |548783                        
|Total Commands Processed                             |42920873                      
|Number of Keys                                       |1157                          
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |28                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ISSUED                                           |116                           
|VMs ARRIVED                                          |113                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |113                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |7                             
|AI RESERVATIONS                                      |0                             
|AIs ISSUED                                           |29                            
|AIs ARRIVED                                          |28                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |28                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |1                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |494                           

 ---------------- VMC-WIDE COUNTERS ----------------
|9ADC8244-8C1F-5728-8581-9D8825A87B6E (simzone_a) VM RESERVATIONS|0                             
|662A8BAF-6270-52A0-B49B-7E1326CCC8B6 (simzone_c) VM RESERVATIONS|0                             
|8E167E6D-B4E1-531D-A90A-A469EF1067C6 (simzone_b) VM RESERVATIONS|0                             
|B37F024B-5D05-5130-8E9E-0804A1122684 (simzone_d) VM RESERVATIONS|0                             

